sdk/tests/conformance/extensions/ext-sRGB.html

```diff
@@ -311,7 +311,7 @@
   var formatString;
   var validFormat;
   switch (format) {
-    case ext.SRGB_EXT: formatString = "sRGB"; validFormat = false; break;
+    case ext.SRGB_EXT: formatString = "sRGB"; validFormat = true; break;
```

Comment:
SRGB is not color renderable, so framebuffer will not be COMPLETE.  It is correct to set validFormat to false.